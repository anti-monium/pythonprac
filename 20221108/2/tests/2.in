((0,0), (0,0), (0,0))
ghjk
(1,), (1,2), (1,3)
111 234 567 890 11 23
((2,3), (3,-3), (-2,-1))
