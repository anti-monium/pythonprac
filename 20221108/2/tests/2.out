Not a triangle
Invalid input
Invalid input
Invalid input
14.00
